source/gulpfile.js

```diff
@@ -1,84 +1,46 @@
 'use strict';
 
 var gulp = require('gulp');
-var rename = require('gulp-rename');
 var eslint = require('gulp-eslint');
-var minify = require('gulp-minify');
 var del = require('del');
 var plumber = require('gulp-plumber');
 
 var globalConfig = {
   scripts_src_dir: 'js',
-  scripts_min_dir: '../build/js',
   build_dir: '../build'
 };
 
-/*
- *
- * JS files build task.
- *
- * Copies and minifies your JS files to build/js/
- *
- */
-gulp.task('js:build', function() {
-  return gulp.src(globalConfig.scripts_src_dir + '/**/*.js')
-    .pipe(plumber())
-    .pipe(rename({dirname: ''}))
-    .pipe(minify({
-      noSource: true
```

Comment:
@hstandaert, can you try this with "compress: false"?
See https://www.npmjs.com/package/gulp-minify

Reply:
`compress: false` results in the minification/obfuscation being skipped entirely.. Could be useful when checking for environments, but not for our use case 😓